roaring.go

```diff
@@ -185,7 +185,7 @@ func (rb *RoaringBitmap) Equals(o interface{}) bool {
 // Add the integer x to the bitmap
 func (rb *RoaringBitmap) Add(x uint32) {
 	hb := highbits(x)
-	ra := rb.highlowcontainer
+	ra := &rb.highlowcontainer
```

Comment:
Take a reference to the highlowcontainer to avoid copying the struct to the local variable. Speeds up things a little.